packages/blocks/src/page-block/utils/bind-hotkey.ts

```diff
@@ -205,10 +211,47 @@ export function bindHotkeys(
     LEFT,
     RIGHT,
     ENTER,
+    TAB,
   } = HOTKEYS;
 
   bindCommonHotkey(page);
 
+  hotkey.addListener(TAB, () => {
+    page.captureSync();
+    for (const block of selection.state.selectedBlocks) {
+      const model = getModelByElement(block);
+      handleIndent(page, model, 0, false);
+    }
+
+    const cachedSelectedBlocks = selection.state.selectedBlocks.concat();
+    requestAnimationFrame(() => {
+      const selectBlocks: DefaultPageBlockComponent[] = [];
+      cachedSelectedBlocks.forEach(block => {
+        const newBlock = document.querySelector(
+          `[data-block-id="${(block as DefaultPageBlockComponent).model.id}"]`
+        );
```

Comment:
The type cast is wrong since `DefaultPageBlockComponent` only corresponds to the `affine-default-page` component, I think `Element` is enough in this scenario

<img width="553" alt="Screenshot 2023-02-08 at 5 35 17 PM" src="https://user-images.githubusercontent.com/18554747/217491357-27afc02d-c4c8-44cf-825a-19ca8f16fddd.png">


```suggestion
        const newBlock = getBlockElementByModel(block.model)
```